GenerateApis.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+# Generates a set of APIs specified by arguments. The APIs can be
+# specified as JSON files, or as a single file containing a list of
+# JSON files, using the "@" prefix.
+#
+# Common examples:
+#
+#   In autorelease.sh:
+#   ./GenerateApis.sh @tmp/ApisToGenerate.txt
+#
+#   When testing a generator:
+#   ./GenerateApis.sh DiscoveryJson/*.json
+
+set -e
+
+if [[ $1 == "" ]]
+then
+  echo "Usage options:"
+  echo "  GenerateApis.sh <path/to/discovery.json> [...]"
+  echo "  GenerateApis.sh @file-with-paths"
+  exit 1
+fi
+
+source ./CSharpGeneratorFunctions.sh
+# install_csharp_generator
```

Comment:
tiny nit:
```suggestion
# install_csharp_generator
source ./CSharpGeneratorFunctions.sh
```

Reply:
More than a nit - install_csharp_generator is a function that I commented out because it was taking a while when testing :) Whoops!